src/backend/distributed/planner/multi_explain.c

```diff
@@ -182,6 +182,8 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 		return;
 	}
 
+	ExplainOpenGroup("Distributed Query", NULL, true, es);
```

Comment:
If you think of it as JSON, true starts an object with {, false starts an array with [. There can be multiple Tasks, so it should start with [. A Task is an object, so it should start with {.

If the second parameter is set, it becomes a field, like "Job" : { .. }. If it is NULL, it generates an unnamed object {..}, which you can use in an array.